components/gitpod-protocol/src/protocol.ts

```diff
@@ -119,6 +119,7 @@ export type IDESettings = {
     defaultIde?: string
     useDesktopIde?: boolean
     defaultDesktopIde?: string
+    useLatestVersion?: boolean
```

Comment:
`useLatestDesktopVersion`

Maybe we should make it even JB specific.

Reply:
After recent discussions about unifying preferences to have a check box fro all IDES, it is indeed makes sense to me.